dask/dataframe/tests/test_io.py

```diff
@@ -295,7 +295,7 @@ def test_compression_multiple_files():
 
 def test_empty_csv_file():
     with filetext('a,b') as fn:
-        df = dd.read_csv(fn)
+        df = dd.read_csv(fn, header=0)
```

Comment:
This was a bug that was masked by pandas implicitly casting. The test was actually asserting dask has different behavior than pandas - this fixes it so the two remain compatible.